cfgrammar/src/lib/yacc/parser.rs

```diff
@@ -55,7 +55,12 @@ impl Error for YaccParserError {}
 
 impl fmt::Display for YaccParserError {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        let s = match self.kind {
+        write!(f, "{}", self.kind)
+    }
+}
```

Comment:
Missing blank line.

Reply:
Should be fixed in https://github.com/softdevteam/grmtools/pull/307/commits/e3770abb08029f1f5e54609c4283fce28cb6d4b5